bin/specextract

```diff
@@ -429,18 +428,35 @@ def determine_rmf_tool(infile,rmffile):
         
         v3(f"Found the following P2_RESP file(s): {caldb_p2_resp_file}")
         
-        v1("Using mkacisrmf...\n")
+        v2("Using mkacisrmf...\n")
 
         rmftool = "mkacisrmf"
 
     return rmftool
 
 
 
+def get_rmf_tool(args):
+    key_id = args["key_id"]
+    instrument = args["instrument"]
+    phafile = args["phafile"]
+    evtfile = args["filename"].split("/")[-1]
+    
+    if instrument == "ACIS":
+        rmffile_ccd = args["rmffile_ccd"]
+        rmftool = _determine_rmf_tool(phafile,rmffile_ccd,evtfile)
+
+        return {key_id : {"rmftool" : rmftool}}
+
```

Comment:
This should handle the not-an-ACIS instrument case. Either erroring out or returning `None` or `{}` or something.

I'd also suggest handing this as a special case, so you have

```python
    if instrument != "ACIS":
        return-something-or-error-out

    rmffile_ccd = args["rmffile_ccd"]
    rmftool = _determine_rmf_tool(phafile,rmffile_ccd,evtfile)
    return {key_id : {"rmftool" : rmftool}}
```